app/Exceptions/Handler.php

```diff
@@ -1,19 +1,22 @@
 <?php
 
+declare(strict_types=1);
+
 namespace App\Exceptions;
 
 use Illuminate\Foundation\Exceptions\Handler as ExceptionHandler;
+use Illuminate\Http\Response;
 use Throwable;
 
 class Handler extends ExceptionHandler
 {
+    use HandlerExceptions;
     /**
      * A list of the exception types that are not reported.
      *
      * @var array
      */
     protected $dontReport = [
```

Comment:
I think `$dontReport` is empty in `Illuminate\Foundation\Exceptions\Handler`, so you can remove it from here.